api/agent/agent_test.go

```diff
@@ -566,8 +513,6 @@ func TestHTTPWithoutContentLengthWorks(t *testing.T) {
 func TestGetCallReturnsResourceImpossibility(t *testing.T) {
 	call := &models.Call{
 		AppID:       id.New().String(),
-		AppName:     "appName",
-		Path:        "/yoyo",
```

Comment:
getting a call without an `FnID` set should be an error, this test shouldn't change that (we should add another test for that), but one should need to be supplied here